### DISCUSSION TITLE
Feature Request - Unread Messages banner

### DISCUSSION THREAD

**Discussion by jlubeck** [flyerhq/flutter_chat_ui#94] [Ideas & Requests]
Usually whenever you leave a chat app and come back later, you get a banner that shows you what was the last message you read so you know where to start reading from.

Example:
<img width="634" alt="Screen Shot 2021-05-20 at 10 19 46 AM" src="https://user-images.githubusercontent.com/3067603/118995140-e7a9fa00-b954-11eb-8816-2e24c19670f6.png">

**Comment by leeyisoft** [comment]
This is a very useful feature

**Reply by demchenkoalex** [reply]
Yep, it is easy to implement too (design-wise), however how to actually make it work.. discussion is here https://github.com/flyerhq/flutter_chat_ui/issues/179
